CondCore/Utilities/test/BuildFile.xml

```diff
@@ -11,3 +11,8 @@
 
 <bin file="testPngHistograms.cpp" name="testPngHistograms">
 </bin>
+
+<bin file="testDriver.cpp" name="testGetPayloadData">
+  <flags TEST_RUNNER_ARGS="/bin/bash CondCore/Utilities/test test_getPayloadData.sh"/>
+  <use name="FWCore/Utilities"/>
+</bin>
```

Comment:
@mmusich , there is no need to have `testDriver.cpp` in order to run test. Just replace the above section with
```
<test name="testGetPayloadData" command="test_getPayloadData.sh"/>
```
remove `testDriver.cpp` and make sure `test_getPayloadData.sh` has executable mode set

Reply:
@smuzaffar, done. Thanks